docs/creation/static-constructor.md

```diff
@@ -0,0 +1,83 @@
+---
+hide:
+  - toc
+---
+
+# Static Constructor
+
+## Problem
+
+You want to construct an object or return a meaningful error message. Unfortunately, in Pony there's no way to do that. Pony constructors always return an initialized instance of their class unless, the constructor is partial in which case nothing is return as we jump to the nearest error handler.
```

Comment:
As written these first two sentences read like there is no way in Pony to "construct an object or return a meaningful error message" -- which is not the case.

Reply:
The additions you made addressed my concern.